packages/integration-tests/src/test-e2e/create_amplify.test.ts

```diff
@@ -8,204 +8,366 @@ import assert from 'assert';
 import { glob } from 'glob';
 import { testConcurrencyLevel } from './test_concurrency.js';
 
-void describe(
-  'create-amplify script',
-  { concurrency: testConcurrencyLevel },
-  () => {
-    before(async () => {
-      // start a local npm proxy and publish the current codebase to the proxy
-      await execa('npm', ['run', 'clean:npm-proxy'], { stdio: 'inherit' });
-      await execa('npm', ['run', 'vend'], { stdio: 'inherit' });
-
-      // nuke the npx cache to ensure we are installing packages from the npm proxy
-      const { stdout } = await execa('npm', ['config', 'get', 'cache']);
-      const npxCacheLocation = path.join(stdout.toString().trim(), '_npx');
-
-      if (existsSync(npxCacheLocation)) {
-        await fs.rm(npxCacheLocation, { recursive: true });
-      }
+type PackageManagerExecutable = 'npm' | 'yarn' | 'yarn-stable' | 'pnpm';
+
+const concurrency = process.env.PACKAGE_MANAGER_EXECUTABLE?.startsWith('yarn')
+  ? 1
+  : testConcurrencyLevel;
```

Comment:
Remove the concurrency for `yarn` since it doesn't support cache for call in parallel https://github.com/yarnpkg/yarn/issues/8854